DEPATISnet.js

```diff
@@ -203,34 +193,33 @@ function scrape(doc, url) {
 
 	newItem.attachments.push({
 		title: "DEPATISnet patent record",
-		document: doc
+		url: url,
+		snapshot: false
 	});
 	
-	var pages = ZU.xpathText(doc, '//table[@class="ergebnis"]/caption[1]');
-	pages = pages.match(/(Seiten|Pages):\s*([0-9]+)/)[2];
-
-	if (pages > 0) {
-		var pdfurl = "https://register.dpma.de/pdfschrift/"+pn+".pdf?dpi=300";
-		
-		newItem.attachments.push({
-			title: newItem.patentNumber,
-			url: pdfurl,
-			mimeType: "application/pdf"
-		});
+	var pages = ZU.xpathText(doc, '//div[@id="inhalt"]/h2');
+	if (pages) {
```

Comment:
```suggestion
	if (pages  && pages.search(/(Seiten|Pages):\s*[0-9]+/) != -1)`) {
```

Reply:
How about this for added robustness? Everything else looks good

Reply:
Okay, but let me rewrite this into the test below. Then we don't need to evaluate this regex twice.